src/ui/components/shared/Forms/TextInput.module.css

```diff
@@ -0,0 +1,49 @@
+.textInput {
+  text-align: left;
+  display: block;
+  width: 100%;
+  border-radius: 0.375rem;
+  border-color: transparent;
+  padding: 0.5rem;
+  outline: 0;
+}
+
+.textInput:focus {
+  box-shadow: 0 0 0 3px var(--primary-accent);
+}
+
+.textCenter {
+  text-align: center;
+}
+
+.default .textInput {
+  background-color: var(--theme-text-field-bgcolor);
+}
+
+.default .textInput:hover {
+  background-color: var(--theme-text-field-bgcolor-hover);
+}
+
+/* Dark theme */
+
+.dark .textInput {
+  background-color: #3e434b; /* Dark: --theme-text-field-bgcolor */
```

Comment:
Is it customary to use the explicit color values instead of css vars here?

Reply:
Yeah, unfortunately. If we use a css variable, it will register the current theme. Meaning you'll hit this page in light theme, but the page is fixed to dark theme, so you'll see a light theme on a dark background.

I looked for a way to do something like (var--:dark--theme-text-field-bgcolor) but that's not possible. So we've gotta hard-code it.